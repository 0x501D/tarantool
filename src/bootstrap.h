const unsigned char bootstrap_bin[] = {
 0x53, 0x4e, 0x41, 0x50, 0x0a, 0x30, 0x2e, 0x31, 0x32, 0x0a, 0x0a, 0xd5,
 0xba, 0x0b, 0xab, 0x18, 0x00, 0xce, 0xa2, 0x27, 0x2b, 0x1e, 0xa7, 0x01,
 0x7f, 0x00, 0x00, 0xb8, 0xc3, 0x89, 0x82, 0x00, 0x02, 0x03, 0x01, 0x82,
 0x10, 0xce, 0x00, 0x00, 0x01, 0x10, 0x21, 0x93, 0xa7, 0x76, 0x65, 0x72,
 0x73, 0x69, 0x6f, 0x6e, 0x01, 0x06, 0xd5, 0xba, 0x0b, 0xab, 0x21, 0x00,
 0xce, 0xf2, 0x9e, 0x4e, 0xd2, 0xa7, 0x01, 0x7f, 0x00, 0x00, 0x28, 0xe2,
 0x89, 0x82, 0x00, 0x02, 0x03, 0x02, 0x82, 0x10, 0xce, 0x00, 0x00, 0x01,
 0x18, 0x21, 0x95, 0xcd, 0x01, 0x10, 0x01, 0xa7, 0x5f, 0x73, 0x63, 0x68,
 0x65, 0x6d, 0x61, 0xa5, 0x6d, 0x65, 0x6d, 0x74, 0x78, 0x00, 0xd5, 0xba,
 0x0b, 0xab, 0x20, 0x00, 0xce, 0x39, 0x40, 0x50, 0x31, 0xa7, 0x01, 0x7f,
 0x00, 0x00, 0x28, 0xe2, 0x89, 0x82, 0x00, 0x02, 0x03, 0x03, 0x82, 0x10,
 0xce, 0x00, 0x00, 0x01, 0x18, 0x21, 0x95, 0xcd, 0x01, 0x18, 0x01, 0xa6,
 0x5f, 0x73, 0x70, 0x61, 0x63, 0x65, 0xa5, 0x6d, 0x65, 0x6d, 0x74, 0x78,
 0x00, 0xd5, 0xba, 0x0b, 0xab, 0x20, 0x00, 0xce, 0xe6, 0x8c, 0x1c, 0x2e,
 0xa7, 0x01, 0x7f, 0x00, 0x00, 0x28, 0xe2, 0x89, 0x82, 0x00, 0x02, 0x03,
 0x04, 0x82, 0x10, 0xce, 0x00, 0x00, 0x01, 0x18, 0x21, 0x95, 0xcd, 0x01,
 0x20, 0x01, 0xa6, 0x5f, 0x69, 0x6e, 0x64, 0x65, 0x78, 0xa5, 0x6d, 0x65,
 0x6d, 0x74, 0x78, 0x00, 0xd5, 0xba, 0x0b, 0xab, 0x1f, 0x00, 0xce, 0xb4,
 0x4b, 0x9a, 0x20, 0xa7, 0x01, 0x7f, 0x00, 0x00, 0x28, 0xe2, 0x89, 0x82,
 0x00, 0x02, 0x03, 0x05, 0x82, 0x10, 0xce, 0x00, 0x00, 0x01, 0x18, 0x21,
 0x95, 0xcd, 0x01, 0x28, 0x01, 0xa5, 0x5f, 0x66, 0x75, 0x6e, 0x63, 0xa5,
 0x6d, 0x65, 0x6d, 0x74, 0x78, 0x00, 0xd5, 0xba, 0x0b, 0xab, 0x1f, 0x00,
 0xce, 0x3d, 0x98, 0x15, 0x27, 0xa7, 0x01, 0x7f, 0x00, 0x00, 0x28, 0xe2,
 0x89, 0x82, 0x00, 0x02, 0x03, 0x06, 0x82, 0x10, 0xce, 0x00, 0x00, 0x01,
 0x18, 0x21, 0x95, 0xcd, 0x01, 0x30, 0x01, 0xa5, 0x5f, 0x75, 0x73, 0x65,
 0x72, 0xa5, 0x6d, 0x65, 0x6d, 0x74, 0x78, 0x00, 0xd5, 0xba, 0x0b, 0xab,
 0x1f, 0x00, 0xce, 0x1c, 0x7c, 0xf8, 0xd7, 0xa7, 0x01, 0x7f, 0x00, 0x00,
 0x28, 0xe2, 0x89, 0x82, 0x00, 0x02, 0x03, 0x07, 0x82, 0x10, 0xce, 0x00,
 0x00, 0x01, 0x18, 0x21, 0x95, 0xcd, 0x01, 0x38, 0x01, 0xa5, 0x5f, 0x70,
 0x72, 0x69, 0x76, 0xa5, 0x6d, 0x65, 0x6d, 0x74, 0x78, 0x00, 0xd5, 0xba,
 0x0b, 0xab, 0x22, 0x00, 0xce, 0x66, 0xb7, 0xea, 0x7f, 0xa7, 0x01, 0x7f,
 0x00, 0x00, 0x28, 0xe2, 0x89, 0x82, 0x00, 0x02, 0x03, 0x08, 0x82, 0x10,
 0xce, 0x00, 0x00, 0x01, 0x18, 0x21, 0x95, 0xcd, 0x01, 0x40, 0x01, 0xa8,
 0x5f, 0x63, 0x6c, 0x75, 0x73, 0x74, 0x65, 0x72, 0xa5, 0x6d, 0x65, 0x6d,
 0x74, 0x78, 0x00, 0xd5, 0xba, 0x0b, 0xab, 0x26, 0x00, 0xce, 0x6b, 0xb2,
 0x24, 0xcb, 0xa7, 0x01, 0x7f, 0x00, 0x00, 0xc8, 0x12, 0x8b, 0x82, 0x00,
 0x02, 0x03, 0x09, 0x82, 0x10, 0xce, 0x00, 0x00, 0x01, 0x20, 0x21, 0x98,
 0xcd, 0x01, 0x10, 0x00, 0xa7, 0x70, 0x72, 0x69, 0x6d, 0x61, 0x72, 0x79,
 0xa4, 0x74, 0x72, 0x65, 0x65, 0x01, 0x01, 0x00, 0xa3, 0x73, 0x74, 0x72,
 0xd5, 0xba, 0x0b, 0xab, 0x26, 0x00, 0xce, 0x2d, 0x5b, 0x36, 0xed, 0xa7,
 0x01, 0x7f, 0x00, 0x00, 0xc8, 0x12, 0x8b, 0x82, 0x00, 0x02, 0x03, 0x0a,
 0x82, 0x10, 0xce, 0x00, 0x00, 0x01, 0x20, 0x21, 0x98, 0xcd, 0x01, 0x18,
 0x00, 0xa7, 0x70, 0x72, 0x69, 0x6d, 0x61, 0x72, 0x79, 0xa4, 0x74, 0x72,
 0x65, 0x65, 0x01, 0x01, 0x00, 0xa3, 0x6e, 0x75, 0x6d, 0xd5, 0xba, 0x0b,
 0xab, 0x24, 0x00, 0xce, 0x60, 0x16, 0xe6, 0x5b, 0xa7, 0x01, 0x7f, 0x00,
 0x00, 0xc8, 0x12, 0x8b, 0x82, 0x00, 0x02, 0x03, 0x0b, 0x82, 0x10, 0xce,
 0x00, 0x00, 0x01, 0x20, 0x21, 0x98, 0xcd, 0x01, 0x18, 0x01, 0xa5, 0x6f,
 0x77, 0x6e, 0x65, 0x72, 0xa4, 0x74, 0x72, 0x65, 0x65, 0x00, 0x01, 0x01,
 0xa3, 0x6e, 0x75, 0x6d, 0xd5, 0xba, 0x0b, 0xab, 0x23, 0x00, 0xce, 0x01,
 0x85, 0x35, 0x93, 0xa7, 0x01, 0x7f, 0x00, 0x00, 0xc8, 0x12, 0x8b, 0x82,
 0x00, 0x02, 0x03, 0x0c, 0x82, 0x10, 0xce, 0x00, 0x00, 0x01, 0x20, 0x21,
 0x98, 0xcd, 0x01, 0x18, 0x02, 0xa4, 0x6e, 0x61, 0x6d, 0x65, 0xa4, 0x74,
 0x72, 0x65, 0x65, 0x01, 0x01, 0x02, 0xa3, 0x73, 0x74, 0x72, 0xd5, 0xba,
 0x0b, 0xab, 0x2b, 0x00, 0xce, 0xcb, 0x64, 0xaa, 0x2d, 0xa7, 0x01, 0x7f,
 0x00, 0x00, 0xc8, 0x12, 0x8b, 0x82, 0x00, 0x02, 0x03, 0x0d, 0x82, 0x10,
 0xce, 0x00, 0x00, 0x01, 0x20, 0x21, 0x9a, 0xcd, 0x01, 0x20, 0x00, 0xa7,
 0x70, 0x72, 0x69, 0x6d, 0x61, 0x72, 0x79, 0xa4, 0x74, 0x72, 0x65, 0x65,
 0x01, 0x02, 0x00, 0xa3, 0x6e, 0x75, 0x6d, 0x01, 0xa3, 0x6e, 0x75, 0x6d,
 0xd5, 0xba, 0x0b, 0xab, 0x28, 0x00, 0xce, 0x87, 0x13, 0x4b, 0xfe, 0xa7,
 0x01, 0x7f, 0x00, 0x00, 0xc8, 0x12, 0x8b, 0x82, 0x00, 0x02, 0x03, 0x0e,
 0x82, 0x10, 0xce, 0x00, 0x00, 0x01, 0x20, 0x21, 0x9a, 0xcd, 0x01, 0x20,
 0x02, 0xa4, 0x6e, 0x61, 0x6d, 0x65, 0xa4, 0x74, 0x72, 0x65, 0x65, 0x01,
 0x02, 0x00, 0xa3, 0x6e, 0x75, 0x6d, 0x02, 0xa3, 0x73, 0x74, 0x72, 0xd5,
 0xba, 0x0b, 0xab, 0x26, 0x00, 0xce, 0xdd, 0x78, 0x58, 0xe7, 0xa7, 0x01,
 0x7f, 0x00, 0x00, 0xc8, 0x12, 0x8b, 0x82, 0x00, 0x02, 0x03, 0x0f, 0x82,
 0x10, 0xce, 0x00, 0x00, 0x01, 0x20, 0x21, 0x98, 0xcd, 0x01, 0x28, 0x00,
 0xa7, 0x70, 0x72, 0x69, 0x6d, 0x61, 0x72, 0x79, 0xa4, 0x74, 0x72, 0x65,
 0x65, 0x01, 0x01, 0x00, 0xa3, 0x6e, 0x75, 0x6d, 0xd5, 0xba, 0x0b, 0xab,
 0x24, 0x00, 0xce, 0x94, 0x62, 0xd1, 0x49, 0xa7, 0x01, 0x7f, 0x00, 0x00,
 0xc8, 0x12, 0x8b, 0x82, 0x00, 0x02, 0x03, 0x10, 0x82, 0x10, 0xce, 0x00,
 0x00, 0x01, 0x20, 0x21, 0x98, 0xcd, 0x01, 0x28, 0x01, 0xa5, 0x6f, 0x77,
 0x6e, 0x65, 0x72, 0xa4, 0x74, 0x72, 0x65, 0x65, 0x00, 0x01, 0x01, 0xa3,
 0x6e, 0x75, 0x6d, 0xd5, 0xba, 0x0b, 0xab, 0x23, 0x00, 0xce, 0x14, 0x67,
 0x27, 0x6b, 0xa7, 0x01, 0x7f, 0x00, 0x00, 0xc8, 0x12, 0x8b, 0x82, 0x00,
 0x02, 0x03, 0x11, 0x82, 0x10, 0xce, 0x00, 0x00, 0x01, 0x20, 0x21, 0x98,
 0xcd, 0x01, 0x28, 0x02, 0xa4, 0x6e, 0x61, 0x6d, 0x65, 0xa4, 0x74, 0x72,
 0x65, 0x65, 0x01, 0x01, 0x02, 0xa3, 0x73, 0x74, 0x72, 0xd5, 0xba, 0x0b,
 0xab, 0x26, 0x00, 0xce, 0x6a, 0x9b, 0x6c, 0xe8, 0xa7, 0x01, 0x7f, 0x00,
 0x00, 0xc8, 0x12, 0x8b, 0x82, 0x00, 0x02, 0x03, 0x12, 0x82, 0x10, 0xce,
 0x00, 0x00, 0x01, 0x20, 0x21, 0x98, 0xcd, 0x01, 0x30, 0x00, 0xa7, 0x70,
 0x72, 0x69, 0x6d, 0x61, 0x72, 0x79, 0xa4, 0x74, 0x72, 0x65, 0x65, 0x01,
 0x01, 0x00, 0xa3, 0x6e, 0x75, 0x6d, 0xd5, 0xba, 0x0b, 0xab, 0x23, 0x00,
 0xce, 0x8c, 0x4d, 0xce, 0x97, 0xa7, 0x01, 0x7f, 0x00, 0x00, 0xc8, 0x12,
 0x8b, 0x82, 0x00, 0x02, 0x03, 0x13, 0x82, 0x10, 0xce, 0x00, 0x00, 0x01,
 0x20, 0x21, 0x98, 0xcd, 0x01, 0x30, 0x02, 0xa4, 0x6e, 0x61, 0x6d, 0x65,
 0xa4, 0x74, 0x72, 0x65, 0x65, 0x01, 0x01, 0x02, 0xa3, 0x73, 0x74, 0x72,
 0xd5, 0xba, 0x0b, 0xab, 0x30, 0x00, 0xce, 0x23, 0x14, 0x27, 0xba, 0xa7,
 0x01, 0x7f, 0x00, 0x00, 0xc8, 0x12, 0x8b, 0x82, 0x00, 0x02, 0x03, 0x14,
 0x82, 0x10, 0xce, 0x00, 0x00, 0x01, 0x20, 0x21, 0x9c, 0xcd, 0x01, 0x38,
 0x00, 0xa7, 0x70, 0x72, 0x69, 0x6d, 0x61, 0x72, 0x79, 0xa4, 0x74, 0x72,
 0x65, 0x65, 0x01, 0x03, 0x01, 0xa3, 0x6e, 0x75, 0x6d, 0x02, 0xa3, 0x73,
 0x74, 0x72, 0x03, 0xa3, 0x6e, 0x75, 0x6d, 0xd5, 0xba, 0x0b, 0xab, 0x24,
 0x00, 0xce, 0x25, 0x54, 0xa2, 0x6d, 0xa7, 0x01, 0x7f, 0x00, 0x00, 0xc8,
 0x12, 0x8b, 0x82, 0x00, 0x02, 0x03, 0x15, 0x82, 0x10, 0xce, 0x00, 0x00,
 0x01, 0x20, 0x21, 0x98, 0xcd, 0x01, 0x38, 0x01, 0xa5, 0x6f, 0x77, 0x6e,
 0x65, 0x72, 0xa4, 0x74, 0x72, 0x65, 0x65, 0x00, 0x01, 0x01, 0xa3, 0x6e,
 0x75, 0x6d, 0xd5, 0xba, 0x0b, 0xab, 0x26, 0x00, 0xce, 0xa6, 0x75, 0x0b,
 0xf4, 0xa7, 0x01, 0x7f, 0x00, 0x00, 0xc8, 0x12, 0x8b, 0x82, 0x00, 0x02,
 0x03, 0x16, 0x82, 0x10, 0xce, 0x00, 0x00, 0x01, 0x20, 0x21, 0x98, 0xcd,
 0x01, 0x40, 0x00, 0xa7, 0x70, 0x72, 0x69, 0x6d, 0x61, 0x72, 0x79, 0xa4,
 0x74, 0x72, 0x65, 0x65, 0x01, 0x01, 0x00, 0xa3, 0x6e, 0x75, 0x6d, 0xd5,
 0xba, 0x0b, 0xab, 0x23, 0x00, 0xce, 0x36, 0x62, 0xa3, 0x7d, 0xa7, 0x01,
 0x7f, 0x00, 0x00, 0xc8, 0x12, 0x8b, 0x82, 0x00, 0x02, 0x03, 0x17, 0x82,
 0x10, 0xce, 0x00, 0x00, 0x01, 0x20, 0x21, 0x98, 0xcd, 0x01, 0x40, 0x01,
 0xa4, 0x75, 0x75, 0x69, 0x64, 0xa4, 0x74, 0x72, 0x65, 0x65, 0x01, 0x01,
 0x01, 0xa3, 0x73, 0x74, 0x72, 0xd5, 0xba, 0x0b, 0xab, 0x16, 0x00, 0xce,
 0x49, 0xee, 0x64, 0xb0, 0xa7, 0x01, 0x7f, 0x00, 0x00, 0x08, 0x1a, 0x8b,
 0x82, 0x00, 0x02, 0x03, 0x18, 0x82, 0x10, 0xce, 0x00, 0x00, 0x01, 0x30,
 0x21, 0x93, 0x00, 0xa0, 0xa5, 0x67, 0x75, 0x65, 0x73, 0x74, 0xd5, 0xba,
 0x0b, 0xab, 0x16, 0x00, 0xce, 0x6d, 0x84, 0xb2, 0x94, 0xa7, 0x01, 0x7f,
 0x00, 0x00, 0x08, 0x1a, 0x8b, 0x82, 0x00, 0x02, 0x03, 0x19, 0x82, 0x10,
 0xce, 0x00, 0x00, 0x01, 0x30, 0x21, 0x93, 0x01, 0xa0, 0xa5, 0x61, 0x64,
 0x6d, 0x69, 0x6e, 0xd5, 0x10, 0xad, 0xed,
};
