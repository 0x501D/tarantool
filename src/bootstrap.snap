SNAP
0.12

лн
